src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  showModal: boolean;
+  setShowModal: (close: boolean) => void;
+  todo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  showModal,
+  setShowModal,
+  todo,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  function getCurrentUser() {
+    getUser(todo.userId).then(setUser);
+  }
+
+  useEffect(() => {
+    getCurrentUser();
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
-    <div className="modal is-active" data-cy="modal">
+    <div
+      className={cn('modal', {
+        'is-active': showModal,
+      })}
+      data-cy="modal"
+    >
       <div className="modal-background" />
 
-      {true ? (
+      {false ? (
```

Comment:
The condition for rendering the `Loader` is hardcoded as `false`. Consider using a state variable to dynamically control when the `Loader` should be displayed, such as when the user data is being fetched.